src/common/src/hash/key.rs

```diff
@@ -758,19 +758,28 @@ mod tests {
         let capacity = 128;
         let seed = 10244021u64;
         let columns = vec![
-            Column::new(seed_rand_array_ref::<BoolArray>(capacity, seed)),
-            Column::new(seed_rand_array_ref::<I16Array>(capacity, seed + 1)),
-            Column::new(seed_rand_array_ref::<I32Array>(capacity, seed + 2)),
-            Column::new(seed_rand_array_ref::<I64Array>(capacity, seed + 3)),
-            Column::new(seed_rand_array_ref::<F32Array>(capacity, seed + 4)),
-            Column::new(seed_rand_array_ref::<F64Array>(capacity, seed + 5)),
-            Column::new(seed_rand_array_ref::<DecimalArray>(capacity, seed + 6)),
-            Column::new(seed_rand_array_ref::<Utf8Array>(capacity, seed + 7)),
-            Column::new(seed_rand_array_ref::<NaiveDateArray>(capacity, seed + 8)),
-            Column::new(seed_rand_array_ref::<NaiveTimeArray>(capacity, seed + 9)),
+            Column::new(seed_rand_array_ref::<BoolArray>(capacity, seed, 0.5)),
+            Column::new(seed_rand_array_ref::<I16Array>(capacity, seed + 1, 0.5)),
+            Column::new(seed_rand_array_ref::<I32Array>(capacity, seed + 2, 0.5)),
+            Column::new(seed_rand_array_ref::<I64Array>(capacity, seed + 3, 0.5)),
+            Column::new(seed_rand_array_ref::<F32Array>(capacity, seed + 4, 0.5)),
+            Column::new(seed_rand_array_ref::<F64Array>(capacity, seed + 5, 0.5)),
+            Column::new(seed_rand_array_ref::<DecimalArray>(capacity, seed + 6, 0.5)),
+            Column::new(seed_rand_array_ref::<Utf8Array>(capacity, seed + 7, 0.5)),
```

Comment:
`0.5` null ratio seems rather high, not sure if realistic 😆 .
I prefer have 2 separate bench, one with nulls (maybe 10% at most?), one without.

Reply:
I see, this is for testing, nevermind ignore it